sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/ng/SqlBase.g4

```diff
@@ -102,6 +102,15 @@ statement
         (PARTITIONED ON identifierList)?
         (TBLPROPERTIES tablePropertyList)? AS query                    #createView
     | ALTER VIEW tableIdentifier AS? query                             #alterViewQuery
+    | ALTER VIEW from=tableIdentifier AS? RENAME TO to=tableIdentifier #renameView
```

Comment:
I don't think we need a dedicated label (i.e. `#renameView`) for this, we can just re-use `#renameTable` for this. That way you don't have to add any code to `SparkSqlParser`. The same goes for all these commands.

We can also integrate this with the `ALTER TABLE ...` equivalent rules (i.e. `ALTER (TABLE|VIEW) ...`. But this is a bit messier, and I'll leave that to you.